src/Raven.Client/Documents/Operations/IOperationProgress.cs

```diff
@@ -41,17 +41,20 @@ public virtual DynamicJsonValue ToJson()
         }
     }
 
-    public class BulkInsertProgress : IOperationProgress
+    public class BulkInsertProgress : IOperationProgress, IOperationProcessedDetails
     {
-        public long Processed { get; set; }
-
+        public long Total { get; set; }
```

Comment:
you changed processed to total. Is it breaking change? @ppekrol ?

Reply:
`Processed` didn't come from `IOperationProgress` so that could be fine. Although can it break binary compatibility?

Reply:
You cannot remove that field. You can mark it as obsolete and set the value from DocumentsProcessed in it instead.

Reply:
done